src/local/butler/scripts/migration/chromium_issue_id_migrator.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Migrate Chromium Issue Ids from Monorail to Buganizer."""
+
+import csv
+
+from google.cloud import ndb
+from clusterfuzz._internal.datastore import data_types
+
+# project_name = 'cluster-fuzz'
+
+def execute(args):
+  """Query Testcases of a project, and update the bug_information
+  or group_bug_information fields to reflect the Buganizer issue
+  id rather than the Monorail issue id."""
+  
+  issue_id_dict = get_monorail_buganizer_issue_id_dictionary()
+
+  for testcase in data_types.Testcase.query(
+      # only target testcases in single project
+      data_types.Testcase.project_name == args.project_name,
+  ):
+    if testcase.bug_information:
+      testcase.bug_information = ndb.StringProperty(issue_id_dict[testcase.bug_information])
+    
+    if testcase.group_bug_information:
+      testcase.group_bug_information = ndb.IntegerProperty(issue_id_dict[testcase.group_bug_information])
+
+    if args.non_dry_run:
+      testcase.put()
```

Comment:
This should be a batch put, to improve performance.

See https://github.com/google/clusterfuzz/blob/4f6d16033bc76a38f15ea51ac34939a9051b68b2/src/local/butler/scripts/migration/jobs_keywords.py#L25 for an example.

The maximum batch size that datastore accepts is 500 at a time.